Makefile

```diff
@@ -98,6 +98,7 @@ endif
 
 DOGFOOD?=0
 NODE_MODULES_SRC?=modules.tar
+NODE_MODULES_CACHEDIR=modules.tar.cachedir
```

Comment:
nit: I prefer to name all our folder name with _ instead of .
modules_tar_cachedir